process/transaction/baseProcess.go

```diff
@@ -138,8 +145,21 @@ func (txProc *baseTxProcessor) checkTxValues(tx *transaction.Transaction, acntSn
 	}
 
 	if stAcc.Balance.Cmp(cost) < 0 {
-		return process.ErrInsufficientFunds
+		return txProc.checkTxFee(tx, stAcc)
 	}
 
 	return nil
 }
+
+func (txProc *baseTxProcessor) checkTxFee(tx *transaction.Transaction, stAcc *state.Account) error {
```

Comment:
if the above comment is implemented this could be deleted. and return insufficient funds directly at line 148.

Reply:
Answered above